### DISCUSSION TITLE
Issue installing on Laravel 8 - Version issue with league/flysystem

### DISCUSSION THREAD

**Discussion by stevegoddard** [spatie/flysystem-dropbox#71] [Q&A]
HI, 
Trying to update to spatie/flysystem-dropbox v2. I've deleted my vendor directory and composer.lock.
My composer.json file is:

    "require": {
        "php": "^7.3|^8.0",
        "doctrine/dbal": "^2.11",
        "fideloper/proxy": "^4.4",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^8.12",
        "laravel/helpers": "^1.3",
        "laravel/tinker": "^2.5",
        "laravel/ui": "^3.0",
        "laravelcollective/html": "^6.0",
        "spatie/laravel-backup": "^7.0",
        "spatie/flysystem-dropbox": "^2.0"
    },

However, I get the following error (in part):

    - spatie/flysystem-dropbox v2.x-dev requires league/flysystem ^2.0.4 -> satisfiable by league/flysystem[2.0.4, 2.0.5, 2.x-dev].
    - You can only install one version of a package, so only one of these can be installed: league/flysystem[1.1.0, ..., 1.x-dev, 2.0.4, 2.0.5, 2.x-dev].
    - laravel/framework 8.x-dev requires league/flysystem ^1.1 -> satisfiable by league/flysystem[1.1.0, ..., 1.x-dev].

It looks to me like there is a clash in the version of league/flysystem required by Laravel vs the one required by this package?
Apologies if I'm misreading this error.

Any advice?

Thanks

**Comment by samuelhgf** [comment]
Here is a solution
https://laracasts.com/discuss/channels/laravel/could-not-installed-spatieflysystem-dropbox-package-on-laravel-8351-version-reason-of-some-dependecies
